src/arm/bbai-bone-buses.dtsi

```diff
@@ -468,7 +468,7 @@ bone_timer_5: &timer15 {
 
 // McASP
 bone_mcasp_0: &mcasp1 {
-        #sound-dai-cells= <0>;
+    #sound-dai-cells= <0>;
```

Comment:
Can you fix the tabs across the document to make these things line up?

Reply:
Believe me, i am trying my best with tabs. Things look pretty here and becomes ugly when pushed 😞

Reply:
There are a number of scripts you could run to format it after. I want to avoid it becoming a bigger mess.